packages/devtools/devtools/src/hooks/useContactBook.ts

```diff
@@ -0,0 +1,22 @@
+//
+// Copyright 2024 DXOS.org
+//
+
+import { useEffect, useState } from 'react';
+
+import { type Contact } from '@dxos/protocols/proto/dxos/client/services';
+import { useClient } from '@dxos/react-client';
+
+export const useContactBook = () => {
```

Comment:
if `client.halo.contacts` works now, I would put this hook in `@dxos/react-client` itself

Reply:
yeah, a very basic (union all identities across all spaces recording which spaces are shared with them) implementation's been there since May